FE/angular/src/app/components/movie-card/movie-card.component.spec.ts

```diff
@@ -2,22 +2,64 @@ import { ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { MovieCardComponent } from './movie-card.component';
 import { GoDetailsComponent } from '../navigation/go-details/go-details.component';
+import { Spectator, createComponentFactory } from '@ngneat/spectator';
 
 describe('MovieCardComponent', () => {
+  let spectator: Spectator<MovieCardComponent>;
   let component: MovieCardComponent;
-  let fixture: ComponentFixture<MovieCardComponent>;
+  const createComponent = createComponentFactory({
+    component: MovieCardComponent,
+    imports: [],
+    declarations: [],
+    providers: [],
+    shallow: true,
+    detectChanges: false
+  });
 
-  beforeEach(async () => {
-    await TestBed.configureTestingModule({
-      declarations: [MovieCardComponent, GoDetailsComponent]
-    }).compileComponents();
+  beforeEach(() => {
+    const mockMovies = {
+      Title: 'Mock Movie',
+      Year: 2000,
+      Rated: 'G',
+      Released: '01 Jan 2000',
+      Runtime: '90 min',
+      Genre: 'Mock Genre',
+      Director: 'Director McMock',
+      Writer: 'Writer Mock, Writer Mockerson',
+      Actors: 'Actor McMock, Actor Mockerson',
+      Plot: 'Mock movie plot summary.',
+      Poster:
+        'https://m.media-amazon.com/images/M/MV5BOTY4YjI2N2MtYmFlMC00ZjcyLTg3YjEtMDQyM2ZjYzQ5YWFkXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_SX300.jpg',
+      imdbID: 'tt123',
+      Type: 'movie'
+    };
 
-    fixture = TestBed.createComponent(MovieCardComponent);
-    component = fixture.componentInstance;
-    fixture.detectChanges();
+    spectator = createComponent({
+      props: {
+        movie: mockMovies,
+        isMovieDetails: false
+      }
+    });
+    component = spectator.component;
   });
 
-  test('should create', () => {
+  test('should create the component', () => {
     expect(component).toBeTruthy();
   });
+
+  test('should set movie Year to "2000"', () => {
+    expect(component.movie.Year).toBe(2000);
+  });
+
+  test('should set isMovieDetails to false', () => {
+    expect(component.isMovieDetails).toBeFalsy();
```

Comment:
This is a fake test. 
When testing UI components, do not check the data values in properties.
Instead check key DOM structures that should be rendered.